### DISCUSSION TITLE
Github Actions failed due to internal failure, how to rerun?

### DISCUSSION THREAD

**Discussion by rogebrd** [community/community#25113] [Actions]
<p>I have a github action that runs whenever we update source code to generate the corresponding web files for our website.  I got a message saying: </p>
<p>.github#L1</p>
<pre><code>GitHub Actions has encountered an internal error when running your job.
</code></pre>
<p>I am wondering how I can remedy this without having to create a dummy push.  Is there a way to manually trigger this action or retry in the case of internal errors?</p>
<p><a href="https://github.com/dropbox/dropbox-api-v2-explorer" rel="nofollow noopener">Repo</a>: </p>
<p><a href="https://github.com/dropbox/dropbox-api-v2-explorer/actions?query=workflow%3A%22Deploy+To+Github+Pages%22" rel="nofollow noopener">Action</a>: </p>
<p><a href="https://github.com/dropbox/dropbox-api-v2-explorer/actions/runs/77544058" rel="nofollow noopener">Error</a>: </p>

**Comment by KyrietS** [comment, marked as answer]
<p>GitHub Actions are currently (2020-04-14 00:37 UTC) degraded: <a href="https://status.github.com" rel="nofollow noopener">status.github.com</a></p>
<p>You can re-run all jobs in case of internal failure. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/c187394c59857838de64a8f26ce3915e680f8be7.png" data-download-href="/uploads/short-url/rC1RMEXgqwRzq8Pknotx2fJK2ur.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/19170699/181088983-40f8350e-7a5f-4d17-8611-2900f0d8939a.png" alt="image.png" data-base62-sha1="rC1RMEXgqwRzq8Pknotx2fJK2ur" width="690" height="309" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/c187394c59857838de64a8f26ce3915e680f8be7_2_690x309.png, https://aws1.discourse-cdn.com/github/original/2X/c/c187394c59857838de64a8f26ce3915e680f8be7.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/c/c187394c59857838de64a8f26ce3915e680f8be7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c187394c59857838de64a8f26ce3915e680f8be7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">930×417 6.45 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by rogebrd** [comment]
<p>Thanks, I must have missed that button on my initial dive into it.  Also, thanks for linking <a href="http://status.github.com" rel="nofollow noopener">status.github.com</a>, will be a useful tool going forward.</p>
